.gen/pipeline/version.go

```diff
@@ -1,3 +1,3 @@
 package pipeline
 
-const PipelineVersion = "0.61.0"
+const PipelineVersion = "sa-role-velero"
```

Comment:
**Reminder**: don't forget to regenerate the files once a valid Pipeline version tag is available.